components/esp_lvgl_port/include/esp_lvgl_port.h

```diff
@@ -12,28 +12,15 @@
 #pragma once
 
 #include "esp_err.h"
-#include "esp_lcd_panel_io.h"
-#include "esp_lcd_panel_ops.h"
 #include "lvgl.h"
-
-#if __has_include ("esp_lcd_touch.h")
-#include "esp_lcd_touch.h"
-#define ESP_LVGL_PORT_TOUCH_COMPONENT 1
-#endif
-
-#if __has_include ("iot_knob.h")
-#include "iot_knob.h"
-#include "iot_button.h"
-#define ESP_LVGL_PORT_KNOB_COMPONENT 1
-#endif
-
-#if __has_include ("iot_button.h")
-#include "iot_button.h"
-#define ESP_LVGL_PORT_BUTTON_COMPONENT 1
-#endif
-
-#if __has_include ("usb/hid_host.h")
-#define ESP_LVGL_PORT_USB_HOST_HID_COMPONENT 1
+#include "esp_lvgl_port_disp.h"
+#include "esp_lvgl_port_touch.h"
+#include "esp_lvgl_port_knob.h"
+#include "esp_lvgl_port_button.h"
+#include "esp_lvgl_port_usbhid.h"
+
+#if LVGL_VERSION_MAJOR == 8
+#include "esp_lvgl_port_compatibility.h"
 #endif
```

Comment:
This must be added to all `esp_lvgl_port_*.h` header files that use the new `lv_display_t` type. (eg. `esp_lvgl_port_disp.h`...)